plugins/performance-lab/includes/site-health/far-future-headers/helper.php

```diff
@@ -0,0 +1,329 @@
+<?php
+/**
+ * Helper function to detect if static assets have far-future expiration headers.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Callback for the far-future caching test.
+ *
+ * @since n.e.x.t
+ * @access private
+ *
+ * @return array{label: string, status: string, badge: array{label: string, color: string}, description: string, actions: string, test: string} Result.
+ */
+function perflab_ffh_assets_test(): array {
+	$result = array(
+		'label'       => __( 'Your site serves static assets with an effective caching strategy', 'performance-lab' ),
+		'status'      => 'good',
+		'badge'       => array(
+			'label' => __( 'Performance', 'performance-lab' ),
+			'color' => 'blue',
+		),
+		'description' => sprintf(
+			'<p>%s</p>',
+			esc_html__(
+				'Serving static assets with far-future expiration headers improves performance by allowing browsers to cache files for a long time, reducing repeated requests.',
+				'performance-lab'
+			)
+		),
+		'actions'     => '',
+		'test'        => 'is_far_future_headers_enabled',
```

Comment:
Can we maybe use `effective_cache_headers` for the name / identifier of this entire functionality? I just find this `far_future_headers` oddly specific, and it sounds strange to me. The purpose for this Site Health is to have "good" caching headers, and I think we should frame the feature as such, rather than the more technical and IMO too specific "far future" terminology.

This applies to all the identifiers using `far_future_headers`, like this function, the Site Health check itself, the directory name etc. I think we should use `effective_cache_headers`.

Reply:
I think we should use `asset` somewhere in the name because this should be differentiated from the effective `Cache-Control` header for page responses so as to not break bfcache (#1807).

So maybe `effective_asset_cache_headers`?

Reply:
@westonruter That works for me!